src/containers/ConceptPage/ConceptForm/ConceptForm.tsx

```diff
@@ -34,16 +33,15 @@ import { NewConceptType, PatchConceptType } from '../../../modules/concept/conce
 import { License, SubjectType, SearchResult, ConceptStatusType } from '../../../interfaces';
 import { ConceptFormType, ConceptFormValues } from '../conceptInterfaces';
 
-interface Props {
-  applicationError: (err: string) => void;
+interface BaseProps {
   concept: ConceptFormType;
   conceptChanged: boolean;
   fetchConceptTags: (input: string, language: string) => Promise<SearchResult>;
-  inModal: boolean;
+  inModal?: boolean;
```

Comment:
Jeg er ikke helt enig i å ha boolean variabler som optional. Jeg tenker at for ts sin skyld burde vi sende eksplisitt den boolske verdien til komponenten.